ExpenseTracker.Api/ExpenseTracker.Application/Services/WalletService.cs

```diff
@@ -72,6 +72,8 @@ public async Task UpdateAsync(UpdateWalletRequest request)
         ArgumentNullException.ThrowIfNull(request);
 
         var wallet = _mapper.Map<Wallet>(request);
+        wallet.OwnerId = _currentUserService.GetUserId();
+        wallet.CreatedBy = _currentUserService.GetUserName();
```

Comment:
I think, wallet owner does not change. And you wrote unnecessary code.
_Createdby_ and _Updatedby_ are always given values ​​when running ### SaveChangesAsync in **AuditInterceptor**